common-graphql/src/main/scala/queries/common/ObsQueriesGQL.scala

```diff
@@ -54,13 +54,7 @@ object ObsQueriesGQL:
                 id
               }
             }
-            target {
-              id
-              sidereal {
-                ra $AngleSubquery
-                dec $AngleSubquery
-              }
-            }
+            target $TargetWithIdSubquery
```

Comment:
The data we need from a target is:

- Name
- Id
- RA
- Dec
- sourceProfile (to calculate SED)

I agree it would be cleaner to only pull this data and put it in the `TargetSummary` (that's actually how I started out). The reason for this solution is:
- I'm using the `Asterism` class to calculate the base coordinates. `Asterism` needs `TargetWithId`'s. I _could_ instead use a list of `Coordinates` from `TargetSummary` targets to calculate  the base coordinates
- Getting the SED needs some (partial optional) data I don't fully understand. Suggestions are welcome if we can make the query smaller